base/linalg/arnoldi.jl

```diff
@@ -318,17 +318,18 @@ iterations derived from [`eigs`](:func:`eigs`).
   as a subtype of `AbstractArray`, e.g., a sparse matrix, or any other type
   supporting the four methods `size(A)`, `eltype(A)`, `A * vector`, and
   `A' * vector`.
-* `nsv`: Number of singular values.
+* `nsv`: Number of singular values. Default: 6.
 * `ritzvec`: If `true`, return the left and right singular vectors `left_sv` and `right_sv`.
    If `false`, omit the singular vectors. Default: `true`.
 * `tol`: tolerance, see [`eigs`](:func:`eigs`).
-* `maxiter`: Maximum number of iterations, see [`eigs`](:func:`eigs`).
+* `maxiter`: Maximum number of iterations, see [`eigs`](:func:`eigs`). Default: 1000.
+* `ncv`: Maximum size of the Krylov subspace, see [`eigs`](:func:`eigs`) (there called `nev`). Default: `2*nsv`.
+* `u0`: Initial guess for the first left Krylov vector. It may have length `m`, the first dimension of `A`, or 0.
+* `v0`: Initial guess for the first right Krylov vector. It may have length `n`, the second dimension of `A`, or 0.
```

Comment:
Needs a `make docs` sync and/or tweaking the rst signature to match again? The commas here make it almost sound like this is a list of 3 allowed lengths, but it's close enough.